setup.py

```diff
@@ -1,20 +1,29 @@
 from setuptools import setup, find_packages
+from pip.req import parse_requirements
+import uuid
+
+setup_requirements = [
+    'pytest-runner'
+]
+
+test_requirements = [
+    'pytest'
+]
+
+install_requirements = [str(ir.req) for ir in parse_requirements(
+                        "requirements.txt", session=uuid.uuid1())]
 
 setup(
     name='binderhub',
-    version='0.1',
-    install_requires=[
-        'kubernetes==3.*',
-        'tornado',
-        'traitlets',
-        'docker',
-        'jinja2',
-        'prometheus_client'
-    ],
+    version='0.1.1',
```

Comment:
is the change of version from 0.1 to 0.1.1 on purpose or typo? Just double checking

Reply:
I bumped it intentionally, but I can change it back if necessary. I usually try to version bump when I file PRs that change some functionality in case people are depending on the existing behavior.

Reply:
since we haven't made a releas of binder yet, this should probably be `0.1.0dev`.